erpnext_documentation/www/docs/user/manual/en/stock/quality-inspection.md

```diff
@@ -30,16 +30,76 @@ Before creating and using a Quality Inspection, it is advised that you do the fo
+
+E.g. checking if the color is white in a color quality check, Yes/No values for certain parameters, etc.
+
+For Non-numeric checks enable the 'Non-numeric' checkbox. You will notice a field **Acceptance Criteria Value** and a section **Value Based Inspection** are visible.
+
+You can simply enter the field Reading Value. The Acceptance Criteria Value can be set in the Quality Inspection Template once and be simply fetched into the Quality Inspection.
+
+<img class="screenshot" alt="Non-numeric Quality Check" src="{{docs_base_url}}/assets/img/stock/quality-inspection-non-numeric-reading.png">
+
+If the Reading Value does not match the Acceptance Criteria Value, the status on the row will be set to 'Rejected' automatically on Save.
+
+### 3.3 Formula Based Quality Checks
+Formula Based Quality Checks are useful for more complex scenarios where just specifying a range or an acceptance value is not enough.
+
+E.g. checking if the grade of a material is A/B/C, checking if the mean of some readings is in a certain range, etc.
+
+Formula Based Quality Checks are applicable to Numeric and Non-numeric Quality Checks.
+
+Enable the 'Formula Based Criteria' checkbox to perform a Formula Based Quality Check. You will then notice a field called **Acceptance Criteria Formula** where you can specify a formula that determines whether a certain check is Accepted or Rejected.
+This formula can be set in the Quality Inspection Template once and be simply fetched into the Quality Inspection.
 
 <img class="screenshot" alt="Acceptance Criteria Formula" src="{{docs_base_url}}/assets/img/stock/acceptance-criteria-formula.png">
 
-This formula depends on the many Reading fields in the Readings table. It can be set manually or via a template.
+This formula depends on the many Reading fields in the Readings table.
+
+For Numeric readings, `reading_1`, `reading_2` and so on are accepted in the formula.
+
+For Non-numeric readings, only `reading_value` is accepted in the formula.
+
+Here are some examples of formulas:
+```py
+# Numeric
+(reading_1 + reading_2) < 10 # sum of both readings is less than 10
+(reading_1 + reading_2) <= 10 # sum of both readings is less than or equal to 10
+mean < 15  # mean of non empty numeric readings is less than 15
+(reading_1 * 2) < 20 # Reading 1 multiplied by 2 is less than 20
+(reading_1) / 2 < 20 # Reading 1 divided by 2 is less than 20
+
+# Non-numeric
+reading_value in ("A", "B", "C") # Reading Value is either A / B / C
+reading_value != "Red" # Reading Value is not equal to Red
+```
+Update the readings and Save. The Status field in the Readings table rows is set automatically based on the formula for acceptance.
+
+### 3.3 Manual Inspection
+
+So far all the Quality Checks have automatic acceptance/rejection on Save. In the real world, there could be cases where a check is rejected but yet will be accepted because there is some tolerance.
```

Comment:
```suggestion
So far, all the Quality Checks have automatic acceptance/rejection on Save. In the real world, there could be cases where a check is rejected but yet will be accepted because there is some tolerance.
```